src/app/vault/password-generator.component.html

```diff
@@ -27,12 +27,16 @@
                         </button>
                     </div>
                     <div class="box-content condensed" [hidden]="!showOptions">
-                        <div class="box-content-row" appBoxRow>
-                            <label for="type" class="sr-only">{{'type' | i18n}}</label>
```

Comment:
I didn't mean to get rid of this `<label>` altogether, we should keep it for screen readers. 
Just change it to:
`<label class="sr-only radio-header">{{'type' | i18n}}</label>`

Reply:
Added back the label.